main.js

```diff
@@ -35,4 +35,165 @@ menuCon.addEventListener('click', () => {
   mobileNav.classList.remove('active');
   background.classList.remove('blur');
   backgroundNav.classList.remove('blur');
-});
\ No newline at end of file
+});
+
+const projectSection = [
+  {
+    name: 'Tonic',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    featuredImage: './images/Snapshoot-Portfolio-3.png',
+    technologies: ['html', 'css', 'javascript'],
+    liveversion: '',
+    source: '',
+    category: ['CANOPY', 'Back End Dev', '2015'],
+  },
+  {
+    name: 'Tonic',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    featuredImage: './images/Snapshoot-Portfolio-3.png',
+    technologies: ['html', 'css', 'javascript'],
+    liveversion: '',
+    source: '',
+    category: ['CANOPY', 'Back End Dev', '2015'],
+  },
+  {
+    name: 'Tonic',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    featuredImage: './images/Snapshoot-Portfolio-4.png',
+    technologies: ['html', 'css', 'Bootstrap'],
+    liveversion: '',
+    source: '',
+    category: ['CANOPY', 'Back End Dev', '2015'],
+  },
+  {
+    name: 'Tonic',
+    description: 'Lorem ipsum dolor sit amet consectetur adipisicing elit.',
+    featuredImage: './images/Snapshoot-Portfolio-5.png',
+    technologies: ['html', 'css', 'React'],
+    liveversion: '',
+    source: '',
+    category: ['CANOPY', 'Back End Dev', '2015'],
+  },
+];
+
+const workSection = document.querySelector('#works-section');
+
+for (let i = 0; i < 4; i += 1) {
+  const subsection = document.createElement('section');
+  subsection.className = 'subsection-one';
+  subsection.innerHTML = `
+  <div class="second-section-mobile-img">
+  <img src="${projectSection[i].featuredImage}" alt="Featured Image" class="responsive">
+  </div>
+  <div class="desktop-img">
+      <img src="${projectSection[i].featuredImage}" alt="" class="responsive">
+  </div>`;
+  const firstSection = document.createElement('div');
+  firstSection.innerHTML = `
+  <div>
+  <h3>
+      ${projectSection[i].name}
+  </h3>
+  </div>`;
```

Comment:
- [ ] Kindly ensure that you follow the Figma template to a fault. Right now, your Project cards are not similar to the design in the Figma template as you can see below
> Yours
![image](https://user-images.githubusercontent.com/90556221/169507391-769f16ba-00b7-4029-9e97-b9f5416f83f2.png)

> Figma 
![image](https://user-images.githubusercontent.com/90556221/169506875-812691ae-707e-4004-932f-cb2786949198.png)

 To highlight, there are unnecessary empty spaces, the description is not aligned properly, font weights are different,  the image is too small and the technologies have different colors and background compared to the Figma template. Kindly implement these changes

- [ ] Your project cards in the mobile section seem to be overflowing their containers, kindly fix this bug and ensure you follow the Figma styling guidelines

![image](https://user-images.githubusercontent.com/90556221/169510111-b69445f1-3228-41ff-be42-5a4961b00c88.png)